src/lwc/psElevateTokenHandler/psElevateTokenHandler.js

```diff
@@ -238,19 +234,20 @@ class psElevateTokenHandler {
             };
         });
 
-        iframe.contentWindow.postMessage(
-            {
-                action: MOUNT_IFRAME_EVENT_ACTION,
-                paymentMethod: paymentMethod
-            },
-            this._visualforceOrigin
-        );
+        const message = {
+            action: MOUNT_IFRAME_EVENT_ACTION,
+            paymentMethod: paymentMethod
+        };
+        this.sendIframeMessage(iframe, message, this._visualforceOrigin);
 
         return mountPromise;
     }
-}
 
-/**
- * A new instance of the Elevate tokenization handler
- */
-export default new psElevateTokenHandler();
```

Comment:
Intentionally converted this away from a singleton, any time this component is referenced we can safely `new` it up. Nobody shares references to it.